Source/Examples/ExampleLibrary/Series/FinancialSeries/CandleStickAndVolumeSeriesExamples.cs

```diff
@@ -17,42 +17,101 @@ namespace ExampleLibrary
     [Tags("Series")]
     public static class CandleStickAndVolumeSeriesExamples
     {
-        [Example("Candles + Volume (combined volume)")]
-        public static Example CombinedVolume()
+        [Example("Candles + Volume (combined volume), adjusting Y-axis")]
+        public static Example CombinedVolume_Adj()
```

Comment:
do not use abbreviations in .NET: https://msdn.microsoft.com/en-us/library/141e06ef%28v=vs.71%29.aspx
I also try to follow this rule on unit tests